src/app/workspace/error.tsx

```
@@ -0,0 +1,33 @@
+"use client";
+
+import { useEffect } from "react";
+
+export default function Error({
```

Biome (2.4.14)

[error] Do not shadow the global "Error" property.

(lint/suspicious/noShadowRestrictedNames)